compiler/src/dotty/tools/dotc/interactive/Interactive.scala

```diff
@@ -131,7 +131,7 @@ object Interactive {
    *  @param includeReferences  If true, include references and not just definitions
    */
   def namedTrees(trees: List[SourceTree], includeReferences: Boolean, treePredicate: NameTree => Boolean)
-    (implicit ctx: Context): List[SourceTree] = {
+    (implicit ctx: Context): List[SourceTree] = safely {
```

Comment:
It's a bit of a shame that this means we'll discard all results when the problem might be localized. Alternatively, we could move the try/catch around `traverseChildren` I think.